statechart_demo_purchase/demo/statechart.xml

```diff
@@ -79,4 +80,35 @@ statechart:
         </field>
     </record>
 
+    <record model="statechart" id="statechart_grandparent_demo">
+        <field name="name">test.inherit.grand.parent</field>
+        <field name="model_ids" eval="[(4, ref('statechart_demo_purchase.model_test_inherit_grand_parent'))]"/>
+        <field name="yaml">
+statechart:
+  name: test.inherit.grand.parent
+  root state:
+    name: root
+    initial: draft
+    states:
+      - name: draft
+        transitions:
+          - event: button_grand_parent
+        </field>
+    </record>
+
+    <record model="statechart" id="statechart_child2_demo">
+        <field name="name">test.inherit.child2</field>
+        <field name="model_ids" eval="[(4, ref('statechart_demo_purchase.model_test_inherit_child2'))]"/>
```

Comment:
What should happen when a subclass redefines the statechart of a parent class?

I suppose it should replace it completely. The developer then being responsible to ensure the statechart of the subclass is compatible with the statechart of the parent.

Reply:
I also suppose that it should replace it completely.